<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtDLGDetailInfoClass</class>
 <widget class="QWidget" name="qtDLGDetailInfoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>302</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>[ Nanomite ] - Detail Information</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtDLGNanomite.qrc">
    <normaloff>:/DLGNanomite/Icons/IDI_MAIN.ico</normaloff>:/DLGNanomite/Icons/IDI_MAIN.ico</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabDetailView">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabProcessDetails">
       <attribute name="title">
        <string>Process Details</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>851</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="layoutPID">
         <item>
          <widget class="QTableWidget" name="tblPIDs">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>15</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>15</number>
           </attribute>
           <column>
            <property name="text">
             <string>PID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>EntryPoint</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ExitCode</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>FilePath</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabThreadDetails">
       <attribute name="title">
        <string>Thread Details</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>851</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="layoutTID">
         <item>
          <widget class="QTableWidget" name="tblTIDs">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>15</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>15</number>
           </attribute>
           <column>
            <property name="text">
             <string>PID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>TID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>EntryPoint</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ExitCode</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>State</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabExceptionDetails">
       <attribute name="title">
        <string>Exception Details</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>851</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="layoutException">
         <item>
          <widget class="QTableWidget" name="tblExceptions">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>15</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>15</number>
           </attribute>
           <column>
            <property name="text">
             <string>Offset</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Exc. Code</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>PID / TID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Module.Function</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabModuleDetails">
       <attribute name="title">
        <string>Module Details</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>851</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="layoutModule">
         <item>
          <widget class="QTableWidget" name="tblModules">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>15</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>15</number>
           </attribute>
           <column>
            <property name="text">
             <string>PID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Modulebase</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>State</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Path</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="qtDLGNanomite.qrc"/>
 </resources>
 <connections/>
</ui>
